operations/app/terraform/vars/demo/locals.tf

```diff
@@ -3,16 +3,19 @@ locals {
   is_temp_env = true
   address_id  = var.address_id
   init = {
-    environment           = var.environment
-    location              = "eastus"
-    is_metabase_env       = false
-    random_id             = local.random_id
-    resource_group_name   = "prime-data-hub-${var.environment}"
-    resource_prefix       = "pdh${var.environment}"
-    okta_redirect_url     = "https://${var.environment}.prime.cdc.gov/download"
-    okta_base_url         = "hhs-prime.oktapreview.com"
-    OKTA_scope            = "simple_report_dev"
+    environment          = var.environment
+    location             = "eastus"
+    is_metabase_env      = false
+    random_id            = local.random_id
+    resource_group_name  = "prime-data-hub-${var.environment}"
+    resource_prefix      = "pdh${var.environment}"
+    okta_redirect_url    = "https://${var.environment}.prime.cdc.gov/download"
+    okta_base_url        = "hhs-prime.oktapreview.com"
+    OKTA_scope           = "simple_report_dev"
     sftp_container_module = false
+    RS_okta_base_url     = "reportstream.oktapreview.com"
+    RS_okta_redirect_url = "https://prime-data-hub-XXXXXXX.azurefd.net/download"
+    RS_OKTA_scope        = "simple_report_dev"
```

Comment:
`RS_OKTA_scope` is `"reportstream_dev"`

Reply:
RS_OKTA_scope has been updated in the latest commit